runtime/doc/options.txt

```diff
@@ -6347,8 +6201,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	mapping: ":inoremap # X^H#", where ^H is entered with CTRL-V CTRL-H.
 	When using the ">>" command, lines starting with '#' are not shifted
 	right.
-	NOTE: 'smartindent' is reset when 'compatible' is set.  When 'paste'
-	is set smart indenting is disabled.
+	NOTE: When 'paste' is set smart indenting is disabled.
```

Comment:
The variable 'smartindent' is untouched while 'paste' is set, hence `smart indenting` being used.